coordinator/src/test/scala/filodb.coordinator/client/SerializationSpec.scala

```diff
@@ -254,4 +258,28 @@ class SerializationSpec extends ActorTest(SerializationSpecConfig.getNewSystem)
 
   }
 
+  it ("should serialize and deserialize result involving Metadata") {
+
+    val keysMap = Map(UTF8Str("ignore") -> UTF8Str("ignore"))
+    val key = CustomRangeVectorKey(keysMap)
+    val cols = Seq(ColumnInfo("value", ColumnType.MapColumn))
+
+    val response: Set[Map[UTF8Str, UTF8Str]] = new mutable.LinkedHashSet[Map[UTF8Str, UTF8Str]]()
+    response += Map((UTF8Str("dc"), UTF8Str("newark")), (UTF8Str("partition"), UTF8Str("1")),
```

Comment:
You can use `"dc".utf8` etc. for brevity.  Anyways this will have to be redone for the partition key column thing.